### PR TITLE
Adds 0xBTC to token list 

### PR DESCRIPTION
**Pull request by admazzola**
**Description**
Adds 0xBTC to token list 

**Additional context**
Contact info:  admin@0xbitcoin.org  

discord.0xbtc.io   

**Metadata**

### CODE REVIEW

**Comment by elenadimitrova** [comment]
We'll need to implement a custom token for this as the StandardToken has fixed 18 decimals.

File: optimism.tokenlist.json [hunk 1]
```diff
@@ -179,6 +179,50 @@
...
+      "symbol": "0xBTC",
+      "decimals": 8,
+      "logoURI": "https://ethereum-optimism.github.io/logos/0xBTC.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x99C9fc46f92E8a1c0deC1b1747d010903E884bE1"
+      }
+    },
+    {
+      "chainId": 10,
+      "address": "0x78eb06535651115e45853409aC24763932e24C06",
```
Comment (elenadimitrova) [inline optimism.tokenlist.json:218]: Please update to newly deployed token at 0xe0BB0D3DE8c10976511e5030cA403dBf4c25165B

File: optimism.tokenlist.json [hunk 2]
```diff
@@ -179,6 +179,50 @@
...
+      "symbol": "0xBTC",
+      "decimals": 8,
+      "logoURI": "https://ethereum-optimism.github.io/logos/0xBTC.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x22F24361D548e5FaAfb36d1437839f080363982B"
+      }
+    },
+    {
+      "chainId": 69,
+      "address": "0x4D772843C34E84dC8091cCe1484623445fBe6eC7",
```
Comment (elenadimitrova) [inline optimism.tokenlist.json:196]: Please update to newly deployed test token with 8 decimals here 0x56B4F5f44d348EC3F07Bf1f31A3B566E5304BedE

File: optimism.tokenlist.json [hunk 3]
```diff
@@ -179,6 +179,50 @@
       "extensions": {
         "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
       }
+    },
+     {
+      "chainId": 42,
+      "address": "0x24a19ee5a5c8757acdebe542a9436d9c796d1c9e",
```
[the remark this answers is not in the document]
Comment (admazzola) [reply optimism.tokenlist.json:185]: yes using the CPU miner software. Or i  can send you some that I mine.

File: optimism.tokenlist.json (hunk 2 above)
Comment (admazzola) [reply optimism.tokenlist.json:196]: I am confused.  so the newly deployed test token address is 0x56B4F5f44d348EC3F07Bf1f31A3B566E5304BedE? or you want a new token deployed....

File: optimism.tokenlist.json (hunk 1 above)
Comment (admazzola) [reply optimism.tokenlist.json:218]: I am confused. so the newly deployed test token address is 0xe0BB0D3DE8c10976511e5030cA403dBf4c25165B? or you want a new token deployed....

**Pull request closed without merging**